doc/user_guide/electron_holography.rst

```diff
@@ -0,0 +1,26 @@
+Electron Holography

```

Comment:
Yes, I think we should do this. This section is now gone (@woozey, keep this in mind when we come to #1079 later).